### ISSUE TITLE
No new devices have been found

### ISSUE THREAD

**Issue by sandercox** [athombv/com.plugwise.adam#4]
Trying to add Anna to my homey.... can access the smile gateway over same wifi as my homey but when trying to add it (new 2.0 version of the app) I get, "No new devices have been found"

**Comment by bckp** [comment]
App find thru name, so if you rename smile like i did in Google Wifi, app will not find it....

**Comment by synegic** [comment]
This seems to be related to mDNS discovery not picking up the gateway devices. I also experienced this a couple of times while developing. A restart of the app usually helps in such case. In the latest additions to #6 I've tried to improve discovery. Maybe this will also work for you.

If above does not help, maybe you have something in your network preventing mDNS broadcasts from coming through. Like multicast snooping/filtering.
